covid19.occ-pla.net/manifest.json

```diff
@@ -35,7 +35,7 @@
     "environment": "occ-covid19",
     "hostname": "covid19.occ-pla.net",
     "revproxy_arn": "arn:aws:acm:us-east-1:504226487987:certificate/f1172750-0c4c-42d7-b051-b927054810af",
-    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/covid19-datadictionary/2.3.7/schema.json",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/covid19-datadictionary/3.8.12/schema.json",
```

Comment:
The tests are failing because of this:
`[  ERROR] sower block is missing in cdis-manifest`
I think adding an empty `sower: []` (see [here](https://github.com/uc-cdis/cdis-manifest/blob/d75f24f20e7ad639641221625fa23da1e4a8661d/chicagoland.pandemicresponsecommons.org/manifest.json#L131)) will be enough

Reply:
Thanks. Added. I'll commit it with the change to be made for gaTrackingId :)